brianlong/www.validators.app
# 183792092 merge leaders with map

#### What's this PR do?
- Moves leaders component to map

#### How should this be manually tested?
- On localhost run leaders daemon for selected network (eg. `rails r daemons/leader_stats_mainnet_update_daemon.rb`)
- Go to homepage and check the map

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket
](https://www.pivotaltracker.com/story/show/183792092)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
